lib/src/services/jira.ts

```diff
@@ -442,12 +447,26 @@ export const mostRecentIssueComment = (
  * @returns the most recent worklog, or undefined if no work has been logged.
  */
 export const mostRecentIssueWorklog = (
-  issue: Issue
+  issue: Issue,
+  issues: readonly Issue[]
```

Comment:
Update the typedoc to include `issues`. Consider renaming `issues` so that we know more obviously what the purpose of the array of issues is in comparison to the single issue.

Reply:
Also check on the other functions where you've added a parameter, so that all of their documentation is up to date.